src/frontend/components/Input/InputTrace/InputTrace.tsx

```diff
@@ -89,6 +89,7 @@ export const InputTrace = ({
     <svg
       ref={svgRef}
       width={'100%'}
+      height={'100%'}
```

Comment:
What was the issue that needed you to update the styles for Input?
Do you need help with that? Happy to take a look. The input specifically was a little tricky to try to get it to be responsive so you're probably battling with that i suspect.

Reply:
Currently the bars look pretty chonky and there's a gap to the right of the wheel:

<img width="611" height="131" alt="Image" src="https://github.com/user-attachments/assets/1fcf2571-0692-4792-a4d2-ac4756fa5124" />